aemedge/blocks/table/table.css

```diff
@@ -100,99 +100,99 @@
   width: 100%;
   overflow-x: auto;
 
-table {
-  width: 100%;
-  max-width: 100%;
-  border-collapse: collapse;
-  font-size: var(--body-font-size-xs);
-
-  th, td {
-    padding: 0 6px;
-  }
+  table {
+    width: 100%;
+    max-width: 100%;
+    border-collapse: collapse;
+    font-size: var(--body-font-size-xs);
+
+    th, td {
+      padding: 0 6px;
+    }
 
-  thead tr {
-    border-top: 2px solid;
-    border-bottom: 2px solid;
-  }
+    thead tr {
+      border-top: 2px solid;
+      border-bottom: 2px solid;
+    }
 
-  td:not([data-align="right"], [data-align="justify"], [data-align="justify"]) {
-    text-align: center;
-  }
+    td:not([data-align="right"], [data-align="justify"], [data-align="justify"]) {
+      text-align: center;
+    }
   }
 }
 
 /* Full column-align (left, center, right) styles for up to 6 columns */
 .column-left-1 tr:not(.single-column-row) > *:first-child {
-    text-align: left;
-  }
```

Comment:
You can consolidate these by comma-separating them. 

```
.column-left-1 tr:not(.single-column-row) > *:first-child,
.column-left-2 table tr > *:nth-child(2),
.column-left-3 table tr > *:nth-child(3),
...
.column-left-6 table tr > *:nth-child(6) {
text-align: left;
}
```

and so on for right and center.